### PR TITLE
[TS SDK] SDK doc improvements

### PR DESCRIPTION
**Pull request by 0xmaayan**
### Description
Some SDK files were out of date and info was missing, this PR
- adds a LICENSE file
- improves the README file
- improves the CONTRIBUTING file
- removes redundant note from the CHANGELOG file

### CODE REVIEW

File: ecosystem/typescript/sdk/README.md [hunk 1]
```diff
@@ -4,130 +4,78 @@
...
+```ts
 cargo run -p aptos -- node run-local-testnet --with-faucet --faucet-port 8081 --force-restart --assume-yes
 ```
 
-Run the SDK tests and make sure they pass. Go to the SDK directory, and setup an env to configure the URLs:
-
-```
-rm .env
-echo 'APTOS_NODE_URL="http://127.0.0.1:8080/v1"' >> .env
-echo 'APTOS_FAUCET_URL="http://127.0.0.1:8081"' >> .env
```
Comment (davidiw) [inline ecosystem/typescript/sdk/README.md:84]: aren't these still kinda needed how does one connect to their local node?

**Review by davidiw — APPROVED** [review]
looks good with one comment.

File: ecosystem/typescript/sdk/README.md (hunk 1 above)
Comment (0xmaayan) [reply ecosystem/typescript/sdk/README.md:84]: yes! and we still have it here https://github.com/aptos-labs/aptos-core/pull/8362/files#diff-5d279b45707e2394d3480bbc09738d4354c0bd20404b6ed78d01c4d7106c58e0R38

File: ecosystem/typescript/sdk/CONTRIBUTING.md [hunk 2]
```diff
@@ -16,3 +16,42 @@
   - Make sure to run `pnpm test` after making changes.
 - Commits
   - Commit messages follow the [Angular convention](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary).
+
+## Creating a pull request
+
+You are welcome to create a pull reuqest against the main branch.
```
Comment (banool) [inline ecosystem/typescript/sdk/CONTRIBUTING.md:22]:
```suggestion
You are welcome to create a pull request against the main branch.
```

**Review by banool — APPROVED** [review]
Awesome, love that the README focuses on the user experience rather than SDK contributor experience.

**Pull request merged**